katcp/kattypes.py

```diff
@@ -707,8 +707,6 @@ def raw_handler(self, msg):
 
     return decorator
 
-# Using partial with no extra parameters lets us 'copy' the function so that we
-# can change the docstring without affecting the original function's docstring
```

Comment:
Because it is no longer true -- the partial call is also applying a
parameter, which makes it the normal use of partial, and therefore not
comment-worthy.

On Wed, Feb 25, 2015 at 9:42 AM, Theuns Alberts <notifications@github.com>
wrote:

> In katcp/kattypes.py
> <https://github.com/ska-sa/katcp-python/pull/72#discussion_r25322716>:
>
> > @@ -707,8 +707,6 @@ def raw_handler(self, msg):
> >
> >      return decorator
> >
> > -# Using partial with no extra parameters lets us 'copy' the function so that we
> > -# can change the docstring without affecting the original function's docstring
>
> Why remove this useful comment?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ska-sa/katcp-python/pull/72/files#r25322716>.
>